cmd/droplet_create.go

```diff
@@ -172,10 +173,11 @@ func (c *DropletCreateCommand) Run(args []string) int {
 		galaxyInstallCmd.Run([]string{})
 
 		c.UI.Info("\nProvisioning server...\n")
-		playbookCmd := ProvisionCmd(environment, "", "")
-		logCmd(playbookCmd, c.UI, true)
 
-		if err := playbookCmd.Run(); err != nil {
+		c.playbook.SetRoot(c.Trellis.Path)
+
+		if err := c.playbook.Run("server.yml", []string{"-e", "env" + environment}); err != nil {
```

Comment:
`env` is missing `=`

Reply:
The Playbook struct is nice because we can easily re-use it in places like this, but I still wonder if we should actually call the `ProvisionCommand` itself here to ensure they're consistent?

```go
provisionCmd := NewProvisionCommand(c.UI, c.Trellis)
provisionCmd.Run([]string{environment})
```

Something like that?

Reply:
Updated in https://github.com/roots/trellis-cli/pull/91/commits/0c2ec858bb93e1f2a3198cabb19ca9896fbcff8b